Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/StartupHandler.java

```diff
@@ -26,18 +26,26 @@ public static void HandleInit(FragmentActivity parent)
     // Ask the user if he wants to enable analytics if we haven't yet.
     Analytics.checkAnalyticsInit(parent);
 
-    String start_file = "";
+    String[] start_files = null;
     Bundle extras = parent.getIntent().getExtras();
     if (extras != null)
     {
-      start_file = extras.getString("AutoStartFile");
+      start_files = extras.getStringArray("AutoStartFiles");
+      if (start_files == null)
+      {
+        String start_file = extras.getString("AutoStartFile");
+        if (!TextUtils.isEmpty(start_file))
+        {
+          start_files = new String[]{start_file};
```

Comment:
Style nitpick (which can be ignored, since I don't think I've seen it written down anywhere): What about some spaces around there?

```suggestion
          start_files = new String[] { start_file };
```

Reply:
Last time I tried that in Java, lint rejected it.

Reply:
Oh, wasn't aware it actually did check that. Disregard this one then (or...perhaps, look into updating the code style? 😉  )